0001-typedb/01-start/03-quickstart.md

```diff
@@ -0,0 +1,673 @@
+        $ac1(accessed-object: $o, valid-action: $a1) isa access;
+        $ac2(accessed-object: $o, valid-action: $a2) isa access;
+        $p1(permitted-subject: $su, permitted-access: $ac1) isa permission;
+        $p2(permitted-subject: $su, permitted-access: $ac2) isa permission;
+    } then {
+        $p1 has validity false;
+    };
+
+rule automatic-permission-validity:
+    when {
+        $p isa permission;
+        not {
+            $p has validity false;
+        };
+    } then {
+        $p has validity true;
+    };
+    
+rule add-view-access:
+    when {
+        $modify isa action, has action-name "modify_file";
+        $view isa action, has action-name "view_file";
+        $ac_modify (accessed-object: $obj, valid-action: $modify) isa access;
+        $ac_view (accessed-object: $obj, valid-action: $view) isa access;
+        (permitted-subject: $subj, permitted-access: $ac_modify) isa permission;
+    } then {
+        (permitted-subject: $subj, permitted-access: $ac_view) isa permission;
+    };
+```
+
+### Upload the schema
+
+To execute the TypeQL statements in the opened file and send them as queries:
+
+1. Ensure the [**Session type**] (schema / data) switch (next to the database dropdown) is set to `schema`.
+2. Ensure the [**Transaction type**] (write / read) switch is set to `write`.
+
+   ![Select transaction type](../../images/quickstart/studio-select-transaction-type.png)
+
+3. Click the green “play” button (middle of the toolbar) to start the transaction.
+4. Click the “checkmark” button (left of the “play” button) to commit the changes.
+
+The transaction has been committed, and `iam` database now has a schema.
+
+The Types panel will now display the entity, relation, and attribute types within a type hierarchy of the schema.
+
+![Types browser with IAM schema](../../images/quickstart/studio-types-browser-iam.png)
+
+Data can now be inserted.
+
+## First important queries
+
+### Read the schema
+
+TypeQL can be used to query the schema.
+
+To execute a simple schema query:
+
+1. Ensure the [**Session type**] (schema / data) switch is set to `schema` (next to the database dropdown).
+2. Ensure the [**Transaction type**] (write / read) switch is set to `read`.
+3. Click the `+` icon right from the [**Project**] panel next to the `iam-schema.tql` tab in the [**Text-editor**] panel.
+4. Copy the TypeQL statement below.
+5. Click the green “play” button.
+
+<!-- test-ignore -->
+```typeql
+match $t sub thing;
+```
+The above query returns all types in the schema and displays the results as a graph.
+
+<div class="note">
+[Warning]
+The `thing` base type will be deprecated in TypeDB version `3.0`. Consider using `entity`, `attribute`, or `relation` 
+base type instead. To produce the same result as the above example, use the following query:
+
+<!-- test-ignore -->
+```typeql
+match $s sub $t; { $t type entity; } or { $t type relation; } or { $t type attribute; };
```

Comment:
This example really does not look very good when I render it in Studio. The structural elegance of our ecosystem might be lost on someone in the face of a graph that looks like this:

<img width="1231" alt="image" src="https://user-images.githubusercontent.com/51956016/231453386-0fa0cd86-5caa-4873-8e65-e9975ef166cd.png">

Reply:
This needs further discussion obviously.

Reply:
Yes, I do believe that it needs further discussion. Probably out of scope for this review tho.  If you want, I can create an issue on the Studio repo.